style.scss

```
@@ -0,0 +1,146 @@
+$white-color: white;
+$blue-color: #2196f7;
+$green-color: #5bbd4b;
+$orchid-color: #862676;
+$gray-color: #cad3d3;
+$grape-color: #282a3f;
+$myFont: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+
+$font-title: 50px;
+$font-item-info: 35px;
+$font-li-sub: 25px;
+$font-li: 17px;
+@mixin layer($top, $left, $sort) {
+  position: absolute;
+  top: $top;
+  left: $left;
+  z-index: $sort;
+}
+@mixin equal-info($font) {
```

```suggestion
@mixin equalInfo($font) {
```
Với tên hàm em đặt tên theo camelCase còn tên biến trong scss đặt theo kecal-case nhé